src/js/App/Sidenav/__snapshots__/NavigationItem.test.js.snap

```diff
@@ -3,13 +3,13 @@
 exports[`NavigationItem parent and itemID with value 1`] = `
 <div>
   <li
-    class="pf-c-nav__item"
+    class="pf-c-nav__item undefined "
     data-ouia-component-id="OUIA-Generated-NavItem-3"
     data-ouia-component-type="PF4/NavItem"
     data-ouia-safe="true"
   >
     <a
-      class="pf-c-nav__link"
+      class="pf-c-nav__link undefined "
```

Comment:
We are adding some undefined classnames here. You could use the `classNames` lib to avoid these.